src/config/env.ts

```diff
@@ -23,7 +23,7 @@ type Transforms<T, K extends keyof T = keyof T> = {
 };
 
 const transforms: Transforms<EnvVars> = {
-	MICROS_ENV: (value?: string) =>  EnvironmentEnum[value || ""] as EnvironmentEnum || EnvironmentEnum.development,
+	MICROS_ENV: (value?: string) =>  EnvironmentEnum[value || ""] as EnvironmentEnum,
```

Comment:
:susp: Curious why we are removing the fallback value?

Reply:
ah good catch I think this is bad resolution because of the casting.  I will fix it